### PR TITLE
New version: JetBrains.CLion.EAP version 222.2680.4 [FP-D]

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Failed
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | CLion (EAP) |     |
| Version     | 222.2680.4     |  |
| Publisher   | JetBrains s.r.o.   |       |
| ProductCode | CLion 222.2270.30          |     |
#### Auto-updated by [vedantmgoyal2009/winget-pkgs-automation](https://github.com/vedantmgoyal2009/vedantmgoyal2009/tree/main/winget-pkgs-automation) in workflow run [1746](<https://github.com/vedantmgoyal2009/vedantmgoyal2009/actions/runs/2390585253>)

### CODE REVIEW

File: manifests/k/KristenMcWilliam/UnitBargainHunter/1.2.0/KristenMcWilliam.UnitBargainHunter.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: KristenMcWilliam.UnitBargainHunter
+PackageVersion: 1.2.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
```
Comment (ItzLevvie) [inline manifests/k/KristenMcWilliam/UnitBargainHunter/1.2.0/KristenMcWilliam.UnitBargainHunter.installer.yaml:7]:
```suggestion
InstallerType: msix
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
cc @Merrit you need to purchase a Code Signing Certificate.

**Comment by Merrit** [comment]
> cc @Merrit you need to purchase a Code Signing Certificate.

I really can't afford hundreds of dollars for a little open-source app.

Is this a new change? It didn't require signing for winget previously.

**Comment by vedantmgoyal2009** [comment]
@merrit I think you had inno installer before. Have you stopped publishing that? I think, it doesn't require any certificate.

**Comment by Merrit** [comment]
> @Merrit I think you had inno installer before. Have you stopped publishing that? I think, it doesn't require any certificate.

Indeed, I switched over to msix since it is much easier to package with - and the required format if I ever wanted to publish on the windows store I believe. :woman_shrugging:

**Comment by vedantmgoyal2009** [comment]
Okay, whenever you publish it to Microsoft Store, then remove the package from the community repository since winget now supports installing from Microsoft Store.

**Comment by Merrit** [comment]
Sounds like a good idea, I will do that then. Thank you :+1:

**Review by ItzLevvie — DISMISSED** [review]

**Comment by Merrit** [comment]
I published on the microsoft store, and the packages were removed from the winget repo as discussed. This can be closed.

Thanks for the advice! :+1: :blue_heart:

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**